spec/components/dashboard_item_component_spec.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DashboardItemComponent, type: :component do
+  let(:dashboard_item_args) { { model: 'Checkouts',
+                                count: 50,
+                                count_term: 'Total checked out',
+                                items: [
+                                  {
+                                    label: 'recalled',
+                                    count: 2
+                                  },
+                                  {
+                                    label: 'overdue',
+                                    count: 3
+                                  }
+                                ] }}
+
+  it 'makes a list' do
+    component = render_inline(described_class, dashboard_item_args).to_html
```

Comment:
could we add component with a `let` like below and remove that each line from the examples?

`let(:component) { render_inline(described_class, dashboard_item_args).to_html }`

I tried it, seemed to be working but I might be missing sth as well? What do you think?

Reply:
yes, great idea. Also, I refactored reduce Mystery Guest